packages/mwp-app-render/src/components/uxcapture/UXCaptureEventMark.jsx

```diff
@@ -0,0 +1,22 @@
+// @flow
+import * as React from 'react';
+
+type Props = {
+	browserEvent: string,
```

Comment:
are all browser events allowed? I think I've only seen reference to `onLoad`, in which case `browserEvent` doesn't need to be passed in.

Reply:
Yeah, that was my question in the description too :)

Reply:
let's just go with always using `onLoad`, then